### PR TITLE
Cleanup in src/compositor/

### PR DESCRIPTION
**Pull request by matthiasbeyer**
This PR does some cleanup in `src/compositor/`.

### CODE REVIEW

**Comment by neithernut** [comment]
We'll probably have PRs which obsolete parts of this one in within the next weeks.

**Comment by matthiasbeyer** [comment]
Well, that's the fancy thing about git: You can have this stuff merged and if parts of it get removed, nobody cares.

I would still like to see these merged, as it improves code quality a bit. And: If parts get obsolete, everything is fine, but other parts which may _not_ get obsolete are cleaned up here as well.

**Comment by neithernut** [comment]
I _do_ know about removing commits. I just wanted to point out that there will probably be a few changes in the compositor backend in the next few weeks.

**Comment by matthiasbeyer** [comment]
So will this be merged?

**Comment by neithernut** [comment]
Can't be merged.

**Comment by matthiasbeyer** [comment]
Rebased and fixed.

**Comment by matthiasbeyer** [comment]
@TheNeikos It's mainly your code which gets a cleanup here. Would like to see an ACK from you. SYN?

**Comment by TheNeikos** [comment]
Sure! But there is lots of things you forgot though ^^

Let these scripts run to see more:

`for i in **.{c,h}; cat $i | grep '.\{81\}'; and echo $i; end`

`for i in **.{c,h}; grep ')\w' $i; and echo $i; end`

(Adapt to your own shell)

**Comment by matthiasbeyer** [comment]
OMFG. But well, I'll do it.

**Comment by TheNeikos** [comment]
:cat:

**Comment by xVin** [comment]
:bird:

**Comment by matthiasbeyer** [comment]
img

**Comment by matthiasbeyer** [comment]
Rebased. @neithernut , @TheNeikos

File: src/compositor/cursor.c [hunk 1]
```diff
@@ -245,16 +243,21 @@ ws_cursor_redraw(
...
 
-    int retval = drmModeSetCursor2(self->cur_fb_dev->fd, self->cur_mon->crtc,
-            self->cursor_fb->handle, w, h, self->x_hp, self->y_hp);
+    int retval = drmModeSetCursor2(self->cur_fb_dev->fd,
+                                    self->cur_mon->crtc,
+                                    self->cursor_fb->handle,
+                                    w,
+                                    h,
+                                    self->x_hp,
+                                    self->y_hp);
```
Comment (neithernut) [inline src/compositor/cursor.c]: This doesn't look right. Also, I'd put `w` and `h` in one line. The last two arguments might also fit into one line.

File: src/compositor/cursor.c [hunk 2]
```diff
@@ -245,16 +243,21 @@ ws_cursor_redraw(
...
         ws_log(&log_ctx, LOG_CRIT, "Could not set cursor");
-        ws_log(&log_ctx, LOG_CRIT, "State was: crtc: %d, handle: %d, "
-                "height: %d, width: %d",
+        ws_log(&log_ctx, LOG_CRIT,
+                "State was: crtc: %d, handle: %d, height: %d, width: %d",
                 self->cur_mon->crtc, self->cursor_fb->handle, w, h);
     }
     retval = drmModeMoveCursor(self->cur_fb_dev->fd, self->cur_mon->crtc,
-            self->x, self->y);
+                                self->x, self->y);
```
Comment (neithernut) [inline src/compositor/cursor.c]: One space too much.

File: src/compositor/cursor.c [hunk 3]
```diff
@@ -290,7 +293,7 @@ ws_cursor_unset(
 ) {
     //<! @todo: Make unsetting work
     int retval = drmModeSetCursor(self->cur_fb_dev->fd, self->cur_mon->crtc,
-            self->cursor_fb->handle, 0, 0);
+                                    self->cursor_fb->handle, 0, 0);
```
Comment (neithernut) [inline src/compositor/cursor.c]: Here, too.

**Comment by matthiasbeyer** [comment]
Rebased again.

**Comment by neithernut** [comment]
ACK.

**Pull request merged**